docs/tables/datadog_logs_metric.md

```diff
@@ -0,0 +1,20 @@
+# Table: datadog_logs_metric
+
+Log-based metrics are a cost-efficient way to summarize log data from the entire ingest stream. This means that even if you use exclusion filters to limit what you store for exploration, you can still visualize trends and anomalies over all of your log data at 10s granularity for 15 months.
+
+With log-based metrics, you can generate a count metric of logs that match a query or a distribution metric of a numeric value contained in the logs, such as request duration.
+
+## Examples
```

Comment:
Can you please add 2-3 more examples around showing information from the `filter_query` and `group_by` columns? From the table doc/column descriptions, it's difficult to tell what the data shape will look like, so examples here can help users figure this out more easily.